user-data-persistence/code/lib/services/airtable.js

```diff
@@ -0,0 +1,139 @@
+/**  
+ * Airtable Implementation of Remote Database CRUD Interface
+ * 
+ * In the remote DB Collection, a UserData Record looks like this:
+ * {
+ *   id: <dbUserId>
+ *   createdTime: <timestamp>
+ *   fields: {
+ *     userIdField: <bixbyUserId>
+ *     userDataField: "{
+ *       <property1>: <bixbyConceptValue1>
+ *       <property2>: <bixbyConceptValue2>
+ *     ..."
+ *   }
+ * }
+ **/
+var http = require('http')
+var properties = require('./../properties.js')
+
+module.exports = {
+  /**
+   * DELETE UserData
+   * params: userData
+   **/
+  deleteUserData: deleteUserData,
+  /**
+   * READ UserData if exists, otherwise return nothing
+   * params: bixbyUserId
+   **/
+  getUserData: getUserData,
+  /**
+   * UPDATE UserData if exists, otherwise CREATE UserData
+   * params: bixbyUserId, userData
+   */
+  putUserData: putUserData
+}
+
+function createUserData(bixbyUserId, userData) {
+  const url = properties.get("config", "baseUrl")
+  const body = {fields:{}}
+  body.fields[properties.get("config", "userIdField")] = bixbyUserId
+  body.fields[properties.get("config", "userDataField")] = JSON.stringify(userData)
+  const options = {
+    format: "json",
+    cacheTime: 0,
+    passAsJson: true,
+    headers: { 
+      Authorization: "Bearer " + properties.get("secret", "apiKey")
+    }
+  }
+  const response = http.postUrl(url, body, options)
+  if (response) {
+    userData = JSON.parse(response.fields[properties.get("config", "userDataField")])
+    userData.$id = response.id
+    return userData
+  }
+}
+
+function deleteUserData(userData) {
+  const dbUserId = userData.$id
+  if (dbUserId) {
+    // Exists. Delete
+    const url = properties.get("config", "baseUrl") + "/" + dbUserId
+    const options = {
+      format: "json",
+      cacheTime: 0,
+      headers: { 
+        Authorization: "Bearer " + properties.get("secret", "apiKey")
+      }
+    }
+    const response = http.deleteUrl(url, {}, options)
+    if (response) {
```

Comment:
ignorable comment :) 
I like being clear on what is the return value but maybe a ternary operation is as clear and more concise? 
`return response ? true : false`